spec/controllers/v1/sessions_controller_spec.rb

```diff
@@ -265,7 +265,7 @@ def expect_logger_msg(level, msg)
       end
 
       context 'routes requiring auth' do
-        %w[mfa verify slo].each do |type|
+        %w[mfa verify].each do |type|
```

Comment:
I think we're reducing test coverage here.  Can we also add a test for the `slo` redirect you've added? would it be similar to the test `it 'redirects on callback from external logout'`?

Reply:
yes it would, ill add a new test case for unauthenticated users issuing an `slo`

Reply:
@erluti just added https://github.com/department-of-veterans-affairs/vets-api/pull/4817/commits/a4521982ecc330a5adec59d20969cac134ce7a94